.circleci/config.yml

```diff
@@ -5120,10 +5111,14 @@ workflows:
             - go-unit-tests
             - ui-unit-tests
 
-      - scan-images-in-quay: #TODO(ROX-10676): Enable scan on every change.
-          <<: *runOnlyOnReleaseAndReleaseCandidateBuilds
+      - scan-images-in-quay:
           context:
           - quay-rhacs-eng-readonly
+          filters:
```

Comment:
I would pull this into a macro since it is used in three places: `scan-images-in-quay`, `retrigger-master-workflow`, and `slack-notify`. Something like this:
```
// runOnMilestones ?
runOnRCsAndNightlies: &runOnRCsAndNightlies
  filters:
    tags:
      only: /.*/
    branches:
      ignore: /.*/
```

Reply:
On the other hand how about removing this filter so we will run scan on every PR?

Reply:
I don't see real value in doing this. On the contrary, it's a lot of resources and we might distract PR authors with irrelevant to their PR information.

Reply:
> we might distract PR authors with irrelevant to their PR information.

+100 to that

I would even go further and lobby for separating image scans into a completely separate workflow, as it's inherently non-hermetic - its result can change from run to run without us touching a single line of code. But this is outside of scope of this PR, and probably not worth changing before migration off CircleCI.